include/libsemigroups/konieczny.hpp

```diff
@@ -237,6 +237,10 @@ namespace libsemigroups {
     using lambda_orb_scc_index_type = typename lambda_orb_type::scc_index_type;
     using rho_orb_index_type        = typename rho_orb_type::index_type;
     using rho_orb_scc_index_type    = typename rho_orb_type::scc_index_type;
+    using rep_info_type             = typename std::tuple<internal_element_type,
```

Comment:
It probably doesn't make much odds, but could you please use an inner struct instead of a tuple here?